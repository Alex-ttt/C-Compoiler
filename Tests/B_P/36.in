int f(int*, char*);

int main()
{
	int a;
	char c;
	f(&a, &c);
}