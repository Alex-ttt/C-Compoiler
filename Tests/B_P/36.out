 Symbols Table:
   f  int function f(
           pointer to int
           pointer to char
  )
   main  int function main(
  )
    {
      Statements:
      f
      (
        &
          a
      ,
        &
          c
      )
      Symbols Table:
              a int
              c char
    }
